components/scream/src/physics/shoc/CMakeLists.txt

```diff
@@ -112,6 +124,8 @@ foreach (SHOC_LIB IN LISTS SHOC_LIBS)
     ${CMAKE_CURRENT_BINARY_DIR}/${SHOC_LIB}_modules
   )
   target_link_libraries(${SHOC_LIB} physics_share scream_share)
+  target_compile_options(${SHOC_LIB} PUBLIC $<$<COMPILE_LANGUAGE:Fortran>:${SCREAM_Fortran_FLAGS}>
```

Comment:
FYI, you don't need to add `SCREAM_Fortran_FLAGS`. They are already added on the `scream_share` lib, and they are a PUBLIC property, so they will transfer to shoc libraries too.

Reply:
thanks